### ISSUE TITLE
Available as artifact?

### ISSUE THREAD

**Issue by tedyoung** [chadlwilson/cloning#22]
Hi. Have you uploaded a release of this fork as an artifact anywhere?

**Comment by chadlwilson** [comment]
Sorry I missed this - wasn't subscribed to notifications on the repo.

Not yet, no, although I did kind-of prepare to do so. Would prefer not to maintain a fork though :-/

Which fix/tweak are you looking for?

**Comment by chadlwilson** [comment]
In any case, I am in a position where I could release this pretty easily. I'm more motivated to do so if it would be useful to, and trusted by others.

**Comment by bilak** [comment]
is there any plan to release? I'd be happy to use fork.

**Comment by chadlwilson** [comment]
Hi @bilak - are there particular (unreleased?) fixes you are looking for in particular?

**Comment by bilak** [comment]
we are migrating to java 17 and I've seen some issues that should be resolved? So I'd be happy to test it with java upgrade.

**Comment by chadlwilson** [comment]
Fair enough. IIRC some of the tweaks made yet not released made it a bit easier to find the place you might be depending on JVM internals (by reducing noise from statics) and resolved a few common ones by adding some more fast cloners to avoid having to do so yourself, but depending on how you are using the Cloner may not resolve all your issues, so temper expectations :)

**Comment by fatom98** [comment]
Are you planning on releasing 1.11?

**Comment by brsmxq** [comment]
@fatom98 here is published version https://github.com/aem-design/cloning and https://search.maven.org/artifact/design.aem/cloning/1.11.1/jar.

**Comment by chadlwilson** [comment]
Cool @brsmxq - thanks for taking this forward further! Hadn't got around to it.

I am happy to coalesce around your fork, hope the cleanups and GH actions work I had done was useful!
